tcl/org.testeditor.tcl.dsl/src/org/testeditor/tcl/dsl/Tcl.xtext

```diff
@@ -183,9 +195,14 @@ StepContentVariable returns tsl::StepContentVariable:
 	value=STRING;
 
 StepContentElement hidden():
-	('<' value=ID '>') |
-	{StepContentElement} ('<>' | '<' WS+ '>');
+	'<' value=ID '>' |
+	{StepContentElement} '<>';
 
 StepContentPunctuation:
-	value=('.'|'?')
-;
\ No newline at end of file
+	DotStepContentPunctuation | BooleanStepContentPunctuation;
```

Comment:
I use boolean step content punctuation in one rule to enforce this punctuation for calls within asserts.